examples/server/server.cpp

```diff
@@ -2611,21 +2638,34 @@ int main(int argc, char ** argv) {
         {"system_info",     llama_print_system_info()},
     });
 
-    httplib::Server svr;
+    std::unique_ptr<httplib::Server> svr;
```

Comment:
Can't you do it without the pointer ?

Reply:
The SSL branch needs to call two different constructors based on the provided arguments. It's ok as proposed